packages/generators/.eslintrc.cjs

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	rules: {
+		'import/no-nodejs-modules': 0,
```

Comment:
I think we have a default eslint config that can be used for Node scripts. It's used here, for example:

https://github.com/Automattic/wp-calypso/blob/c2a3f57266c1bb6123cd9b474491c31f42fca6a9/packages/effective-module-tree/.eslintrc.js#L1-L4